qa/resources/template.conf

```diff
@@ -34,6 +34,7 @@ scorex {
     mcBlockHeight = %(BLOCK_HEIGHT)d
     mcNetwork = %(NETWORK)s
     withdrawalEpochLength = %(WITHDRAWAL_EPOCH_LENGTH)d
+    initialCumulativeCommTreeHash = "d6847c3b30727116b109caa0dae303842cbd26f9041be6cf05072e9f3211a457d6847c3b30727116b109caa0dae303842cbd26f9041be6cf05072e9f3211a457d6847c3b30727116b109caa0dae303842cbd26f9041be6cf05072e9f32110000"
```

Comment:
instead of hard-coding the value we need to get it from genesis info, because it strictly depends on genesis MC block.